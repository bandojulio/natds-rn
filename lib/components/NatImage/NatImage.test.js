var _jsxFileName='lib/components/NatImage/NatImage.test.js';var _react=require('react');var _react2=_interopRequireDefault(_react);
var _reactTestRenderer=require('react-test-renderer');var _reactTestRenderer2=_interopRequireDefault(_reactTestRenderer);
require('jest-styled-components/native');

var _NatImage=require('./NatImage');function _interopRequireDefault(obj){return obj&&obj.__esModule?obj:{default:obj};}

describe('NatImage component default',function(){
it('Should render NatImage component ',function(){
var button=_reactTestRenderer2.default.create(
_react2.default.createElement(_NatImage.NatImage,{__source:{fileName:_jsxFileName,lineNumber:10}})).
toJSON();

expect(button).toMatchSnapshot('NatImage snapshot');
});
});
//# sourceMappingURL=NatImage.test.js.map