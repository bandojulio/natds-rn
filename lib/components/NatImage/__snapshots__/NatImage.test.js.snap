// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NatImage component default Should render NatImage component  1`] = `
<View
  style={
    Array [
      Object {
        "width": "",
      },
    ]
  }
  type="default"
>
  <Image
    source={
      Object {
        "uri": undefined,
      }
    }
    style={
      Array [
        Object {
          "borderRadius": 0,
          "height": "",
          "width": "",
        },
      ]
    }
    type="default"
  />
</View>
`;
